### PR TITLE
Do a monthly release if needed

### PR DESCRIPTION
**Pull request by sbrunner**
The merges from dependabot didn't creates a new release then do it
monthly if needed

### CODE REVIEW

File: .github/workflows/rebuild.yaml [hunk 1]
```diff
@@ -48,6 +48,9 @@ jobs:
       - name: Build
         run: make
 
+      - if: release
```
Comment (arnaud-morvan) [inline .github/workflows/rebuild.yaml:51]: I do not understand where `release` variable comes from.

File: ci/requires_release [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    date_str = version.split(".dev")[1][:8]
+    date_obj = datetime.datetime.strptime(date_str, "%Y%m%d")
+    days = (datetime.datetime.now() - date_obj).days
+    required = "true" if days > 30 else "false"
+
+    print(f"days: {days}")
+    print(f"required: {required}")
+    print(f"##[set-output name=days;]{days}")
+    print(f"##[set-output name=required;]{required}")
```
Comment (arnaud-morvan) [inline ci/requires_release:25]: Not sure about what does this script, does `print(f"##[set-output name=required;]{required}")` really set something in current job, outside current script ?

**Review by arnaud-morvan — COMMENTED** [review]
I'm not sure to understand everything here, would like some explaination, maybe by meet after next daily.

File: .github/workflows/rebuild.yaml (hunk 1 above)
Comment (sbrunner) [reply .github/workflows/rebuild.yaml:51]: id not if ...

File: ci/requires_release (hunk 2 above)
Comment (sbrunner) [reply ci/requires_release:25]: Yes: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions

**Review by arnaud-morvan — APPROVED** [review]
LGTM

**Pull request merged**